core/src/test/java/io/jenkins/pluginhealth/scoring/service/ScoreServiceIT.java

```diff
@@ -143,44 +138,38 @@ void shouldOnlyRetrieveLatestScoreForPlugins() {
         Set.of(p1s, p2s, p1sOld, p2sOld, p1sOld2).forEach(scoreService::save);
         assertThat(scoreRepository.count()).isEqualTo(5);
 
-        final Map<String, Score> summary = scoreService.getLatestScoresSummaryMap();
+        List<Score> scoreList = new ArrayList<>();
+        scoreList.add(p1s);
+        scoreList.add(p2s);
+        scoreList.add(p2sOld);
+        scoreList.add(p1sOld);
+        scoreList.add(p1sOld2);
+
+        final Map<String, Score> summary = scoreService.getLatestScoresSummaryMap(scoreList);
```

Comment:
This shows that we need to provide the scores we want to see to the method, when those scores are in fact in the database.
The method makes no sense to me now.

Reply:
Thanks @alecharp updated

Reply:
Thanks @alecharp updated